src/components/combo/combo.ts

```diff
@@ -441,9 +432,7 @@ export default class IgcComboComponent<T extends object = any>
       this.selectionController.select(this._value as Item<T>[]);
     }
 
-    if (!this.singleSelect) {
-      this.updateValue();
-    }
```

Comment:
Removing this check regresses the single selection search behavior. To see the regression in action in single selection mode type a search term, say "Sofia", press enter to select the matched item. Now while still focused, type "v" to search for "Varna". The letter "v" is not displayed in the input field.

Reply:
Can you re-test it with the latest changes?